### DISCUSSION TITLE
Which source Namida uses for fetching lyrics?

### DISCUSSION THREAD

**Discussion by NikunjK8** [namidaco/namida#159] [General]
Title

**Comment by NikunjK8** [comment]
Also why is lyrics source greyed out?

![Screenshot_2024-04-08-04-43-12-291_com msob7y namida](https://github.com/namidaco/namida/assets/99175083/f1a750da-1cf2-4dcc-b1f0-39409cab7090)

**Reply by MSOB7YY** [reply]
u should enable lyrics first from player screen, the sources are:
  - local: fetches any matching .lrc .txt in the same track directory if found, or the ones in namida cache
  - internet: fetches lrc from `https://lrclib.net` or text based from a google search 
  - auto: does `local`, if not found then `internet`

**Comment by NikunjK8** [comment]
Got it
